package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "surveyor",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "node-sass": "^4.14.1",
+    "react": "^16.13.1",
```

Comment:
@rafayet-monon We can set some packages to install the specific latest version here.
For example here `"react": "16.13.1"`.
Because another collaborator in the in this project will get the same version as your local machine 😁